velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1770,6 +1790,13 @@ template <typename T>
 struct ToISO8601Function {
   VELOX_DEFINE_FUNCTION_TYPES(T);
 
+  ToISO8601Function() {
+    auto formatter =
+        functions::buildJodaDateTimeFormatter("yyyy-MM-dd'T'HH:mm:ss.SSSZZ");
+    VELOX_CHECK(!formatter.hasError(), "Default format should always be valid");
```

Comment:
perhaps, include the actual error into the error message

Reply:
Thanks. Done.